### PR TITLE
Sscs 10286 increase case event date row formatting

### PR DESCRIPTION
**Pull request by dattipoeHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by MohammedHMCTS — APPROVED** [review]

**Review by TeaWithLucas — APPROVED** [review]

**Review by HarryH96 — APPROVED** [review]

File: benefit/data/sheets/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -2495,6 +2495,36 @@
...
+    "CaseTypeID": "Benefit",
+    "ID": "createTestCase",
+    "Name": "Create test case",
+    "Description": "Create a test case in validAppealCreated state",
+    "DisplayOrder": 999,
+    "PostConditionState": "validAppeal",
+    "SecurityClassification": "Public"
+  },
+  {
+      "LiveFrom": "01/01/2018",
```
Comment (wmarshid) [inline benefit/data/sheets/CaseEvent/CaseEvent.json:2509]: minor/optional: indentation

File: benefit/data/sheets/CaseEvent/CaseEvent.json [hunk 2]
```diff
@@ -2495,6 +2495,36 @@
...
+      "CaseTypeID": "Benefit",
+      "ID": "createWithDwpTestCase",
+      "Name": "Create with FTA test test",
+      "Description": "Create a test case in withDwp state",
+      "DisplayOrder": 997,
+      "PostConditionState": "withDwp",
+      "SecurityClassification": "Public"
+  },
+  {
+      "LiveFrom": "01/01/2018",
```
Comment (wmarshid) [inline benefit/data/sheets/CaseEvent/CaseEvent.json:2519]: minor/optional: indentation

File: benefit/data/sheets/CaseEvent/CaseEvent.json [hunk 3]
```diff
@@ -2495,6 +2495,36 @@
     "PostConditionState": "appealCreated",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2018",
```
Comment (wmarshid) [inline benefit/data/sheets/CaseEvent/CaseEvent.json:2499]: minor/optional: should we not change these to the current date as per @MohammedHMCTS suggestion?

File: benefit/data/sheets/CaseEvent/CaseEvent.json [hunk 4]
```diff
@@ -2495,6 +2495,36 @@
...
+    "Description": "Create a test case in validAppealCreated state",
+    "DisplayOrder": 999,
+    "PostConditionState": "validAppeal",
+    "SecurityClassification": "Public"
+  },
+  {
+      "LiveFrom": "01/01/2018",
+      "CaseTypeID": "Benefit",
+      "ID": "createWithDwpTestCase",
+      "Name": "Create with FTA test test",
```
Comment (wmarshid) [inline benefit/data/sheets/CaseEvent/CaseEvent.json:2512]: minor: test test?

**Review by wmarshid — APPROVED** [review]

**Pull request merged**